docs/content/concepts/spec/fileindex.md

```diff
@@ -98,9 +98,71 @@ This class use (64-bits) long hash. Store the num hash function (one integer) an
 
 Define `'file-index.bitmap.columns'`.
 
+Bitmap file index format (V2):
```

Comment:
Document `file-index.bitmap.version`, old version readers can not read this v2 bitmap index, so if the user want to use old version, he/she should know how to configure this.